compiler/noirc_frontend/src/hir/resolution/errors.rs

```diff
@@ -333,7 +333,7 @@ impl<'a> From<&'a ResolverError> for Diagnostic {
                 "Usage of the `#[foreign]` or `#[builtin]` function attributes are not allowed outside of the Noir standard library".into(),
                 ident.span(),
             ),
-            ResolverError::OracleMarkedAsConstrained { ident } => Diagnostic::simple_error(
+            ResolverError::OracleMarkedAsConstrained { ident } => Diagnostic::simple_warning(
```

Comment:
aztec-nr uses many oracle functions, none of which are marked as unconstrained. To ease the transition I'm changing this to a warning for now.